### DISCUSSION TITLE
I created plugin to preview bittorrent files

### DISCUSSION THREAD

**Discussion by kirasok** [sxyazi/yazi#1007] [2. Show and tell]
![show-case](https://github.com/sxyazi/yazi/assets/75790517/f9f18e78-73c4-4827-89fd-3f4a0f93faf1)

It uses transmission-cli under the hood

**source**: https://github.com/kirasok/torrent-preview.yazi

**Comment by sxyazi** [comment]
Thank you for making it, love it!

Do you have plans to add a scrolling feature to it? That way I can continue scrolling down when the content is long enough.

**Reply by kirasok** [reply]
I implemented scrolling in [76970b6](https://github.com/kirasok/torrent-preview.yazi/commit/76970b6f9d6f3031e9cd57c8595a53e9f9f48c18).

Thank you for creating yazi! It's the best terminal file manager

**Reply by sxyazi** [reply]
Sweet, thanks!
